public/js/app/models/Group.js

```diff
@@ -2,10 +2,16 @@ define(["app/app"], function(App) {
   "use strict";
 
   App.Group = DS.Model.extend({
-    identifier: DS.attr('string'),
     username: DS.attr('string'),
+    screenName: DS.attr('string'),
     type: DS.attr('string'),
-    info: DS.attr('string'),
-    rss: DS.attr('string')
+
+    isGroup: function() {
```

Comment:
But in code here we could simply return true for group and return false for user, they won't have their type changed, will they?